lib/entity_cache/entity_cache.rb

```diff
@@ -20,10 +20,9 @@ def self.build(subject, persistent_store: nil, write_behind_delay: nil)
 
     Clock::UTC.configure instance
     Telemetry::Logger.configure instance
-
     Storage::Temporary.configure instance, subject
 
-    Storage::Persistent.configure instance, subject, implementation: persistent_store
```

Comment:
Yeah, I actually kind of felt weird about it, because it was unusual. But colloquially we often use the phrase "vary the implementation," and I can't think of any way of writing this code that resembles that concept more.